packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -424,6 +424,23 @@ export const gliner = (model: ModelData): string[] => [
 model = GLiNER.from_pretrained("${model.id}")`,
 ];
 
+export const indextts = (model: ModelData): string[] => [
+	`# Download model files from ${model.id} into a 'checkpoints' directory
+# e.g., using: huggingface-cli download ${model.id} \
+  bigvgan_discriminator.pth bigvgan_generator.pth bpe.model dvae.pth gpt.pth unigram_12000.vocab \
+  --local-dir checkpoints
+from indextts.infer import IndexTTS
```

Comment:
```suggestion
	`# Download model
from huggingface_hub import snapshot_download

snapshot_download(${model.id}, local_dir="checkpoints")

from indextts.infer import IndexTTS
```

To have a "all-in-one" snippet it's even better to use `snapshot_download` which is the method equivalent to the mentioned CLI.

Reply:
Also instead of listing all files individually, better to just download the full repo. This is more future-proof if you rename files in the future + it doesn't have much drawbacks since other files are just tiny text files (so instant to download).

Reply:
Finally, by having `snapshot_download` in the snippet, it checks that the directory is up to date each time the script is run. If files are outdated, latest version is downloaded. If files are up to date, it is an (almost-)free operation.